gluefactory/scripts/run_training_jpldd.py

```diff
@@ -0,0 +1,165 @@
+"""
```

Comment:
Do we still need this file?

Reply:
Not necessarily, it was just a playground :) Could keep it it case we need to run it outside train.py... But not needed really